src/handwritten/_install_daqmx.py

```diff
@@ -238,14 +372,69 @@ def _install_daqmx_windows_driver() -> None:
 
     """
     installed_version = _get_daqmx_installed_version()
-    download_url, latest_version = _get_driver_details()
+    download_url, latest_version, release, supported_os = _get_driver_details("Windows")
+    if not download_url:
+        raise click.ClickException(f"Failed to fetch the download url.")
+    if not release:
+        raise click.ClickException(f"Failed to fetch the release version string.")
+    else:
+        if installed_version and latest_version:
+            _confirm_and_upgrade_daqmx_driver(
+                latest_version, installed_version, download_url, release
+            )
+        else:
+            _install_daqmx_driver_windows(download_url)
```

Comment:
It is strange how the if-case is OS independent and the else-case is Winodws specific.

Reply:
I guess this is because we split off into OS-specific code at two points:
- installdriver -> _install_daqmx_driver_windows or _install_daqmx_driver_linux
- _confirm_and_upgrade_daqmx_driver -> _install_daqmx_driver_windows_core or _install_daqmx_driver_linux_core

This is because _install_daqmx_driver_windows and _install_daqmx_driver_linux are 90% identical.